visidata/fill.py

```diff
@@ -33,4 +33,4 @@ def _undo():
     vd.status("filled %d values" % n)
 
 
-Sheet.addCommand('f', 'setcol-fill', 'fillNullValues(cursorCol, someSelectedRows)', 'fills null cells in selected rows of current column with contents of non-null cells up the current column')
+Sheet.addCommand('f', 'setcol-fill', 'fillNullValues(cursorCol, someSelectedRows)', 'Fills empty cells in selected rows of current column. This uses the contents of non-null cells up the current column.')
```

Comment:
I see how "empty" is more international English than "null", but "null" has a specific meaning in VisiData: None, or the value of `options.null_value`.  And if this option isn't set to the empty string, then `fill` won't actually fill empty cells.

Reply:
OK. My instinct is to minimise technical terms in documentation, but we can't sacrifice accuracy.